src/ui/Modal.tsx

```diff
@@ -61,7 +67,6 @@ const Title = styled(Dialog.Title, {
 type ModalProps = {
   children: ReactNode;
   title?: string;
-  onClose: () => void;
```

Comment:
should we make `onOpenChange` required? Or is it only needed to overwrite normal functionality?

Reply:
hrm.   maybe we should.  but technically if we're just using a radix trigger to manage the open state, then there is no need for open or onOpenChange